examples/quickstart_aws/quickstart_aws/__init__.py

```diff
@@ -24,8 +25,11 @@
     resources={
         # With this I/O manager in place, your job runs will store data passed between assets
         # on S3 in the location s3://<bucket>/dagster/storage/<asset key>.
-        "io_manager": s3_pickle_io_manager.configured({"s3_bucket": {"env": "S3_BUCKET"}}),
-        "s3": s3_resource,
+        "io_manager": ConfigurablePickledObjectS3IOManager(
+            s3_resource=my_s3_resource,
+            s3_bucket=EnvVar("S3_BUCKET"),
+        ),
+        "s3": my_s3_resource,
```

Comment:
```suggestion
        "s3": S3Resource(),
```
Since instantiating the the S3Resource is just the one line, i think it makes sense to have it right here instead of above the definitions

Reply:
oh i didnt see it in the IO manager. Yeah the way you have it makes the most sense then!